db/migrate/20170111135212_rename_draft_to_published.rb

```diff
@@ -0,0 +1,5 @@
+class RenameDraftToPublished < ActiveRecord::Migration
+  def change
+    rename_column :events, :draft, :published
```

Comment:
Repopulate sample data sounds for me like a database reset. I dont think you want to reset the database with every single commit/merge when running in production mode.